api/Window.json

```diff
@@ -10214,7 +10224,7 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```

Comment:
Nothing else about this API is marked experimental. I suspect this was just missed in previous updates.